styleguide/_themes/derek/scss/snowflakes/stories.scss

```diff
@@ -0,0 +1,94 @@
+.story-colLeft {
+  @include make-md-column(6);
+}
+
+.story-colRight {
+  @include make-md-column(5);
+  @include make-md-column-push(1);
+}
+
+.story-title {
+  text-align: left;
+}
+
+.story-caseStudy-button {
+  @include button;
+  @include button-learning;
+  margin-bottom: 1rem;
+  width: auto;
```

Comment:
why auto? Shouldnt it work like the rest of the site until we make that change globally? Also is there a reason we want the button to have a bottom margin instead of the container just have some bottom padding?